# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../netcat/default-unsigned-arch.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---


This patch fixes arches like PowerPC which do default to unsigned char.

  - Rene Rebe <rene@exactcode.de>

--- netcat-0.6.1/src/flagset.c.orig	2002-10-03 12:25:16.000000000 +0200
+++ netcat-0.6.1/src/flagset.c	2004-06-28 23:00:40.529869256 +0200
@@ -134,7 +134,7 @@
 
 int netcat_flag_count(void)
 {
-  register char c;
+  register signed char c;
   register int i;
   int ret = 0;
 
